src/main/java/com/habicus/core/service/User/UserService.java

```diff
@@ -22,4 +22,33 @@
  */
```

Comment:
Can we add Java Doc to all the public functions in this class?
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html